src/app/body/profile/edit-profile/edit-profile.component.html

```diff
@@ -41,6 +41,20 @@ <h2>Edit Profile</h2>
                                     class="form-control" id="recipient-name" (input)="checkAvailabilityLive()">
                             </div>
                         </div>
+                        <a *ngIf="userAvailable" >
+                            <svg xmlns="http://www.w3.org/2000/svg" style="display: none;">
+                                <symbol id="check-circle-fill" fill="currentColor" viewBox="0 0 16 16">
+                                  <path d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zm-3.97-3.03a.75.75 0 0 0-1.08.022L7.477 9.417 5.384 7.323a.75.75 0 0 0-1.06 1.06L6.97 11.03a.75.75 0 0 0 1.079-.02l3.992-4.99a.75.75 0 0 0-.01-1.05z"/>
+                                </symbol>
+                            </svg>
+                            <div class="alert alert-success d-flex align-items-center" role="alert">
+                                <svg class="bi flex-shrink-0 me-2" width="24" height="24" role="img" aria-label="Success:"><use xlink:href="#check-circle-fill"/></svg>
+                                <div>
+                                    Username available
+                                </div>
+                            </div>
+                        </a>
```

Comment:
why to use svg here

Reply:
it is a green tick here if username is available